lib/libzfs_core/libzfs_core.c

```diff
@@ -340,6 +344,12 @@ lzc_exists(const char *dataset)
 	return (ioctl(g_fd, ZFS_IOC_OBJSET_STATS, &zc) == 0);
 }
 
+int
+lzc_sync(const char *pool_name)
```

Comment:
Force of habit from earlier work. As for doing a `nvlist_t **outnvl` as well, that sounds good.